protoutils/protoutils.go

```diff
@@ -13,6 +13,10 @@ import (
 // InterfaceToMap attempts to coerce an interface into a form acceptable by structpb.NewStruct.
 // Expects a struct or a map-like object.
 func InterfaceToMap(data interface{}) (map[string]interface{}, error) {
+	return interfaceToMapHelper(data, true)
```

Comment:
I'm confused, now it looks like there is a mixture of ignoreOmitEmpty and shouldOmitEmpty. If we're going with ignoreOmitEmpty, then this should be false, correct?